traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -136,3 +151,99 @@ func Read(w http.ResponseWriter, r *http.Request) {
 
 	api.WriteResp(w, r, resolvers)
 }
+
+func Delete(w http.ResponseWriter, r *http.Request) {
+	inf, sysErr, userErr, errCode := api.NewInfo(r, []string{"id"}, []string{"id"})
+	tx := inf.Tx.Tx
+	if sysErr != nil || userErr != nil {
+		api.HandleErr(w, r, tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	userErr, sysErr, statusCode, alert, respObj := delete(inf)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, tx, statusCode, userErr, sysErr)
+		return
+	}
+
+	api.WriteRespAlertObj(w, r, tc.SuccessLevel, alert.Text, respObj)
+}
+
+func delete(inf *api.APIInfo) (error, error, int, tc.Alert, tc.FederationResolver) {
```

Comment:
You mean change it so everything is before the errors or just the alerts and resolver? I'd rather not change the return order of "User Error, System Error, Response Code" because it's semi-standard in our code base.

Reply:
I was suggesting to change it to follow the TO convention of the user error, sys error, and response code being the last three returned values and putting the other two return values at the beginning.

Reply:
ah, well I've done that.